promql-compliance-tester.yml

```diff
@@ -30,6 +35,24 @@ query_tweaks:
   # UNCOMMENT FOR VICTORIAMETRICS:
   #- note: 'VictoriaMetrics incorrectly modifies incoming start/end range query timestamps to be aligned to the resolution step size.'
   #  align_timestamps_to_step: true
+  #
+  # UNCOMMENT FOR NEW RELIC:
+  # - note: 'New Relic is sometimes off by 1ms when parsing floating point start/end timestamps.'
+  #   truncate_timestamps_to_ms: 1000
+  # - note: 'New Relic adds a "prometheus_server" label to distinguish Prometheus servers, leading to extra labels in query results. These need to be stripped before comparisons.'
+  #   no_bug: true
+  #   drop_result_labels:
+  #     - prometheus_server
+  # - note: 'New Relic omits the first resolution step in the output.'
+  #   ignore_first_step: true
+  # - note: 'New Relic stores metric counters as deltas, not cumulative values. New Relic will recognize these as gauges by if "_cnt" is appended to the metric name.'
+  #   query_string_replacements:
+  #     remove: _total
+  #     replacement: _total_cnt
+  #     exemptions:
+  #       - delta
```

Comment:
I think this should be `irate` and not `delta`, as `delta` is meant to be used on gauges, while `irate` is another counter function that is missing in this list.

Btw. `resets` is another counter-only function, but I guess that one will not be able to work anyway as long as NR doesn't store absolute counter values that would allow computing where a counter reset happened.